utils/cpuload/scheddebug/scheddebug.go

```diff
@@ -142,9 +178,52 @@ func (self *SchedReader) allowErrorLogging() bool {
 	return false
 }
 
+func GetSelfCgroup() (string, error) {
```

Comment:
We already get this information at the start of cAdvisor. Worth it to re-use it here?